Tasks/google-play-release/task.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "8cf7cac0-620b-11e5-b4cf-8565e60f4d27",
-    "name": "google-play-release",
+    "name": "GooglePlayRelease",
```

Comment:
Yeah, we should maintain a consistent naming convention across each task within an extension, so I'd prefer not to rename this unless we want to rename all of the tasks.